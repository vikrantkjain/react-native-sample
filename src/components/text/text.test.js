import React from 'react';
import renderer from 'react-test-renderer';
import Text from './index';

test('render correctly', () => {
  const tree = renderer
    .create(<Text style={{color: '#fff'}}>Hello</Text>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
