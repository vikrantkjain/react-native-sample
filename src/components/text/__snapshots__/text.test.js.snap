// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly 1`] = `
<Text
  style={
    [
      {
        "fontFamily": "AvenirNext-Regular",
        "fontSize": 20,
      },
      {
        "color": "#fff",
      },
    ]
  }
>
  Hello
</Text>
`;
